extensions/dependencies/count_vectorizer.html

```diff
@@ -0,0 +1 @@
+<script src="{{cache_slug}}/core/templates/dev/head/domain/classifier/CountVectorizerService.js"></script>
```

Comment:
Does it make sense to bundle some of these individual dependencies together, if they are likely to be used together in any case? Or at least bundle them for now and then only split them later if needed?